libs/vscode/messaging/src/lib/features/running-tasks.ts

```diff
@@ -0,0 +1,38 @@
+import { NotificationType, NotificationType2 } from 'vscode-jsonrpc';
+import {
+  MessagingNotification,
+  MessagingNotification2,
+} from '../messaging-notification';
+import {
+  endRunningTasks,
+  setUpdatingRunningTasks,
+  startRunningTasks,
+  UpdatedRunningTask,
+} from '@nx-console/shared-running-tasks';
+
+export const NxStartedRunningTasks: MessagingNotification<number> = {
+  type: new NotificationType('nx/startedRunningTasks'),
+  handler: (processId) => {
+    console.log('Started running tasks:', processId);
```

Comment:
I think logging on all these events could be very noisy here... we don't want to spam the console or vscode output